pdart/pipeline/StateMachine.py

```diff
@@ -20,12 +20,17 @@
 
 
 class StateMachine(object):
-    def __init__(self, dirs: Directories, proposal_id: int) -> None:
+    def __init__(
+        self,
+        dirs: Directories,
+        proposal_id: int,
+        selected_suffixes: Optional[bool] = False,
```

Comment:
`selected_suffixes` is always `False` in the pipeline, so the parameter is unnecessary.